app_dart/lib/src/request_handlers/get_branches.dart

```diff
@@ -10,48 +10,34 @@ import 'package:meta/meta.dart';
 import '../datastore/cocoon_config.dart';
 import '../foundation/providers.dart';
 import '../foundation/typedefs.dart';
-import '../request_handlers/refresh_github_commits.dart';
+import '../request_handlers/utils.dart';
 import '../request_handling/body.dart';
 import '../request_handling/request_handler.dart';
 
-/// Signature for a function that calculates the backoff duration to wait in
-/// between requests when GitHub responds with an error.
-///
-/// The `attempt` argument is zero-based, so if the first attempt to request
-/// from GitHub fails, and we're backing off before making the second attempt,
-/// the `attempt` argument will be zero.
-typedef GitHubBackoffCalculatorBranch = Duration Function(int attempt);
-
-/// Default backoff calculator.
-@visibleForTesting
-Duration twoSecondLinearBackoffBranch(int attempt) {
-  return const Duration(seconds: 2) * (attempt + 1);
-}
-
-/// Queries GitHub for the list of all available branches, and returns those
+/// Queries GitHub for the list of all available branches on
+/// [flutter/flutter] repo, and returns list of branches
```

Comment:
nit: flutter/flutter isn't a variable. Should it be [config.flutterSlug]?

Reply:
updated.